pkg/proxy/userspace/proxier.go

```diff
@@ -972,6 +1015,13 @@ func (proxier *Proxier) iptablesHostNodePortArgs(nodePort int, protocol api.Prot
 	return args
 }
 
+// Build a slice of iptables args for an from-non-local public-port rule.
+func (proxier *Proxier) iptablesNonLocalNodePortArgs(nodePort int, protocol api.Protocol, proxyIP net.IP, proxyPort int, service proxy.ServicePortName) []string {
+	args := iptablesCommonPortalArgs(nil, false, false, proxyPort, protocol, service)
+	args = append(args, "-m", "state", "--state", "NEW", "-j", "ACCEPT")
```

Comment:
Any chance this PR will get some attention before the code freeze next week?

On Fri, Jan 15, 2016 at 12:55 PM, Tim Hockin <notifications@github.com>
wrote:

> In pkg/proxy/userspace/proxier.go
> <https://github.com/kubernetes/kubernetes/pull/18804#discussion_r49903862>
> :
>
> > @@ -972,6 +1015,13 @@ func (proxier *Proxier) iptablesHostNodePortArgs(nodePort int, protocol api.Prot
> >  	return args
> >  }
> >
> > +// Build a slice of iptables args for an from-non-local public-port rule.
> > +func (proxier *Proxier) iptablesNonLocalNodePortArgs(nodePort int, protocol api.Protocol, proxyIP net.IP, proxyPort int, service proxy.ServicePortName) []string {
> > +	args := iptablesCommonPortalArgs(nil, false, false, proxyPort, protocol, service)
> > +	args = append(args, "-m", "state", "--state", "NEW", "-j", "ACCEPT")
>
> I think this and a TODO for e2e are the only things blocking LGTM
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/18804/files#r49903862>.
>